packages/jest/src/jest-configuration-builder.ts

```diff
@@ -18,17 +18,12 @@ const ARRAY_PROPERTIES_TO_CONCAT = [
  * merge the data as lodash#merge would do it. 
  */
 function concatArrayProperties(objValue: unknown, srcValue: unknown, property: string) {
-  if (!ARRAY_PROPERTIES_TO_CONCAT.includes(property)) {
+  if (!ARRAY_PROPERTIES_TO_CONCAT.includes(property) || !Array.isArray(objValue)) {
```

Comment:
I think of `objValue` is not an array it should throw as well, no? It still means that we're trying to concat a property which is not an array, meaning that we've put it in the list by mistake.

Reply:
I think you are right. But that should be catch with unit testing. So, I will remove that from the code.

Reply:
Updated.